MobileSDK/sdk/src/main/java/org/lantern/mobilesdk/Lantern.java

```diff
@@ -6,26 +6,17 @@
 import com.google.android.gms.analytics.HitBuilders;
 import com.google.android.gms.analytics.Tracker;
 
-
 import java.io.File;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.concurrent.atomic.AtomicBoolean;
 
 /**
- * Created by ox.to.a.cart on 1/28/16.
+ * API for embedding the Lantern proxy
  */
-public class Lantern {
+public abstract class Lantern {
```

Comment:
I went to build a new version of FireTweet that uses the new SDK. I was successful up to the point of trying to sign into Twitter. This page is loaded in a WebView, and we had a special method in the old SDK for configuring WebViews to inherit an app's proxy settings. See https://github.com/getlantern/lantern/blob/lantern-mobile/src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/sdk/Lantern.java#L139 and https://github.com/getlantern/lantern/blob/lantern-mobile/src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/sdk/ProxySettings.java Any possibility we could restore this? I just want to verify running Lantern as a service works in FireTweet.